ccmbioinfo/lfs
# CARDS-2083: Update importTestSql.sql and generate_test_sql.py so that they can be used with the updated Clarity column names

This PR fixes `compose-cluster/mssql/importTestSql.sql` and `compose-cluster/mssql/generate_test_sql.py` so that they can be used for testing the Clarity Importer functionality.

### Testing Instructions

1. Build this (`CARDS-2083`) branch, including a Docker image, with `mvn clean install -Pdocker`.
2. `cd compose-cluster`.
3. Ensure that no other Docker environments are running (`docker ps -a` and `docker volume ls` should both return empty lists)
4. Clean up any previous Docker Compose configuration with `./cleanup.sh`
5. `python3 generate_compose_yaml.py --dev_docker_image --oak_filesystem --composum --adminer --cards_project cards4prems --mssql`
6. `docker-compose build && docker-compose up -d`
7. Navigate to http://localhost:1435/ to access adminer. Login with the following credentials:
    - System: `MS SQL (beta)`
    - Server: `mssql`
    - Username: `sa`
    - Password: `testPassword_`
    - Database: `master`
8. On the left bar, click _Import_, upload the `compose-cluster/mssql/importTestSql.sql` file, and click _Execute_.
9. Visit http://localhost:8080 and login as `admin`:`admin`.
10. In a new tab, visit http://localhost:8080/Subjects.importClarity
11. Refresh the tab showing the main CARDS dashboard.
12. _17_ new _Patients_ should be created - 1 _Patient_ for every SQL row with a valid `EMAIL_ADDRESS` _and_ either `Yes` for `EMAIL_CONSENT_YN` or `Activated` for `MYCHART STATUS`.
13. Stop and clean up everything with `docker-compose down && docker-compose rm && docker volume prune -f && ./cleanup.sh`
14. `cd mssql`
15. `python3 generate_test_sql.py -n 5 ~/testMockClarity.sql`
16. `cd ..`
17. `python3 generate_compose_yaml.py --dev_docker_image --oak_filesystem --composum --adminer --cards_project cards4prems --mssql`
18. `docker-compose build && docker-compose up -d`
19. Navigate to http://localhost:1435/ to access adminer. Login with the following credentials:
    - System: `MS SQL (beta)`
    - Server: `mssql`
    - Username: `sa`
    - Password: `testPassword_`
    - Database: `master`
20. On the left bar, click _Import_, upload the `~/testMockClarity.sql` file, and click _Execute_.
21. Visit http://localhost:8080 and login as `admin`:`admin`.
22. In a new tab, visit http://localhost:8080/Subjects.importClarity
23. Refresh the tab showing the main CARDS dashboard.
24. Review the newly created _Patients_. _Patients_ should only be created for SQL rows with a valid `EMAIL_ADDRESS` _and_ either `Yes` for `EMAIL_CONSENT_YN` or `Activated` for `MYCHART STATUS`.
25. Stop and clean up everything with `docker-compose down && docker-compose rm && docker volume prune -f && ./cleanup.sh`,
